auth_proxy/models/oauth.py

```diff
@@ -21,8 +21,11 @@ class Client(Base):
     client_id = Column(String, primary_key=True)
     client_secret = Column(String, nullable=False)
 
-    _redirect_uris = Column(Text)
-    _default_scopes = Column(Text)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    user = orm.relationship('User')
```

Comment:
I think that's the case here. This is a unidirectional many (clients) to one (user) relationship. Possibly the confusion is coming from the fact that i'm creating a User at the same time I'm creating the Client? I should probably change that so that Clients connect to existing Users instead anyways.